File: java/src/main/java/com/taobao/top/link/endpoint/Message.java
package com.taobao.top.link.endpoint;

import java.util.Map;

// just simple version
public class Message {
	public short protocolVersion = 1;
	public short messageType;

	public int statusCode;
	public String statusPhase;
	public int flag;
	public String token;

	public Map<String, Object> content;
}

File: csharp/src/Taobao.Top.Link/Endpoints/Message.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Taobao.Top.Link.Endpoints
{
    /// <summary>for endpoints talking
    /// </summary>
    public class Message
    {
        public short MessageType { get; set; }
        public int StatusCode { get; set; }
        public string StatusPhase { get; set; }
        public int Flag { get; set; }
        public string Token { get; set; }
        public IDictionary<string, string> Content { get; set; }
    }
}